pycbc/workflow/minifollowups.py

```diff
@@ -652,3 +655,10 @@ def create_noop_node():
     exe.add_pfn(pfn)
     node = wdax.Node(exe)
     return node
+
+def add_wiki_row(outfile, cols):
```

Comment:
Yeah, that was the idea. I had considered putting it in the script itself but thought it might be used by other scripts eventually. If you think it makes more sense in the script, I can move it into `pycbc_sngl_minifollowup` and if it's ever needed again we can move it to somewhere more general.